ivy/functional/backends/paddle/experimental/norms.py

```diff
@@ -28,8 +28,8 @@
 )
 def batch_norm(
     x: paddle.Tensor,
-    mean: paddle.Tensor,
-    variance: paddle.Tensor,
+    running_mean: paddle.Tensor,
```

Comment:
`Positional keywords mean, variance is not allowed in Ivy to be passed as  **args`.  making changes to align with the original paddle function args.